src/hammer-vlsi/tech_test.py

```diff
@@ -652,21 +648,21 @@ def test_twwt_wire(self) -> None:
                 # Check that the resulting spacing is the min spacing
                 self.assertGreaterEqual(s, m.get_spacing_for_width(w))
                 # Check that there is no DRC
-                self.assertGreaterEqual(m.snap(m.pitch * (2*num_tracks + 1)), m.snap(m.min_width + s*3 + w*2))
+                self.assertGreaterEqual(m.pitch * (2 * num_tracks + 1), m.min_width + s*3 + w*2)
                 # Check that if we increase the width slightly we get a DRC violation
                 w = w + (m.grid_unit*2)
                 s = m.get_spacing_for_width(w)
-                self.assertLess(m.snap(m.pitch * (2*num_tracks + 1)), m.snap(m.min_width + s*3 + w*2))
+                self.assertLess(m.pitch * (2 * num_tracks + 1), m.min_width + s*3 + w*2)
 
 
     def test_min_spacing_for_width(self) -> None:
         # Generate multiple stackups, but we'll only use the largest for this test
         stackup = StackupTestHelper.create_test_stackup_list()[-1]
         for m in stackup.metals:
             # generate some widths:
-            for x in [1.0, 2.0, 3.4, 4.5, 5.25, 50.2]:
+            for x in ["1.0", "2.0", "3.4", "4.5", "5.25", "50.2"]:
                 # coerce to a manufacturing grid, this is just a test data point
```

Comment:
Why not use the helper function to "coerce"

Reply:
It's just as clunky, if not more, IMO:
```
for x in ["1.0", "2.0", "3.4", "4.5", "5.25", "50.2"]:
  w = Decimal(x) * m.min_width
# vs.
for x in [1.0, 2.0, 3.4, 4.5, 5.25, 50.2]:
  w = Decimal(coerce_to_grid(x, StackupTestHelper.mfr_grid())) * m.min_width
```

Reply:
ok thats fine then.